block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh

```diff
@@ -44,7 +44,7 @@ function add_worker_pools_and_zones {
     local zones_array=( `bx cs zones |tail -n +3` )
     for zone in "${zones_array[@]}"
     do
-        bx cs worker-pool-create --name pool_$zone --cluster $1 --machine-type $PVG_BX_MACHINE_TYPE --size-per-zone  $PVG_BX_CLUSTER_WORKERS_COUNT
+        bx cs worker-pool-create --name pool_$zone --cluster $1 --flavor $PVG_BX_MACHINE_TYPE --size-per-zone  $PVG_BX_CLUSTER_WORKERS_COUNT
```

Comment:
use `ibmcloud` here as well similar to other places

Reply:
We will not use PVG tester  functions

Reply:
For the VPC the cluster create procedure is entirely different  we will create with subnet and vpc id